includes/Services/ProcessPayment/ApmProcessor.php

```diff
@@ -158,9 +158,7 @@ protected function getAdditionalFields( $exclude = [] ): array {
 			return [];
 		}
 
-		if ( ! is_admin() ) {
-			WC()->cart->calculate_totals();
-		}
+		WC()->cart->calculate_totals();
```

Comment:
I think this was part of a fix from Andrii. @andryes can you please confirm?

Reply:
Same for all cases like this one, for example: `BankAccountProcessor`, `CardProcessor`, `BankAccountBlock`